samples/ImplicitFlow/AuthorizationServer/Startup.cs

```diff
@@ -0,0 +1,156 @@
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using CryptoHelper;
+using OpenIddict;
+using Microsoft.Extensions.Configuration;
+using System;
+using Mvc.Server.Models;
+using Mvc.Server.Services;
+
```

Comment:
When you have a minute, please re-order the usings (System first, then alphabetical order)